web-client/src/address.rs

```diff
@@ -92,9 +92,11 @@ impl Address {
 
 #[wasm_bindgen]
 extern "C" {
-    #[wasm_bindgen(typescript_type = "Address | string")]
+    // #[wasm_bindgen(typescript_type = "Address | string")]
+    #[wasm_bindgen(typescript_type = "string")]
     pub type AddressAnyType;
 
-    #[wasm_bindgen(typescript_type = "(Address | string)[]")]
+    // #[wasm_bindgen(typescript_type = "(Address | string)[]")]
```

Comment:
Maybe remove the comment?

Reply:
Fixed by allowing the class instance as argument instead =)